invenio_query_parser/contrib/elasticsearch/walkers/dsl.py

```diff
@@ -80,6 +81,24 @@ def visit(self, node, op):
     def visit(self, node):
         return node.value
 
+    @visitor(WildcardQuery)
+    def visit(self, node):
+        def query(keyword):
```

Comment:
Panos has now left. Can you maybe guide us on concretely what type of tests we would need to write?